File: app/src/main/java/com/intelegain/agora/model/CustomerData.java
package com.intelegain.agora.model;

import java.util.ArrayList;

/**
 * Created by kiran.p on 3/1/17.
 */

public class CustomerData {

    public String UserMasterID = "", FName = "", LName = "",Email ="",Error = "";

    public ArrayList<CustomerData> customer_login_data = new ArrayList<CustomerData>();
    public int Status = 0;
}

File: app/src/main/java/com/intelegain/agora/model/CustomerData.kt
package com.intelegain.agora.model

import java.util.*

/**
 * Created by kiran.p on 3/1/17.
 */
class CustomerData {
    @kotlin.jvm.JvmField
    var UserMasterID = ""
    @kotlin.jvm.JvmField
    var FName = ""
    @kotlin.jvm.JvmField
    var LName = ""
    @kotlin.jvm.JvmField
    var Email = ""
    @kotlin.jvm.JvmField
    var Error = ""
    @kotlin.jvm.JvmField
    var customer_login_data = ArrayList<CustomerData>()
    var Status = 0
}